napari/layers/points/points.py

```diff
@@ -1934,6 +1975,9 @@ def _paste_data(self):
             self._size = np.append(
                 self.size, deepcopy(self._clipboard['size']), axis=0
             )
+            self._symbol = np.append(
+                self.symbol, deepcopy(self._clipboard['symbol']), axis=0
+            )
```

Comment:
I do not like this, but at this moment do not have a better idea (appending on the end of `np.ndarray` is ineffective).

Reply:
yeah, we talked before about having a buffer to make these operations more performant. Notice however that this is just mirroring the existing behaviour of everything else around it... We should take care of this in a separate PR.